markdown/ch7.markdown

```diff
@@ -649,7 +682,7 @@ for ax in (ax0, ax1):
 What a glorious image! Realise that this artifact was created before color
 photography existed! Notice God's pearly white robes, John's white beard,
 and the white pages of the book held by Prochorus, his scribe — all of which
-were missing from the MSE-based alignment.
+were missing from the MSE-based alignment, but nevertheless look clear.
```

Comment:
The point of the sentence was to say that the MSE, being the wrong metric, failed to produce the white patches correctly, while the NMI alignment worked great.